src/provider/mod.rs

```diff
@@ -404,6 +425,23 @@ impl RpcHandler {
             .map(|(hash, path, size)| ListResponse { hash, path, size });
         futures::stream::iter(items)
     }
+
+    /// Invoke validate on the database and stream out the result
+    fn validate(
+        self,
+        _msg: ValidateRequest,
+    ) -> impl Stream<Item = ValidateResponse> + Send + 'static {
+        self.inner
+            .db
+            .validate(4)
```

Comment:
constant, to make clear what this number means

Reply:
Was meant to replace this with `num_cpus::get()`.

Reply:
not replaced yet